crates/ironrdp-graphics/src/zgfx/mod.rs

```diff
@@ -578,7 +578,7 @@ mod tests {
             0x20, // "The quick brown "
             0x0E, 0x00, 0x00, 0x00, // second segment is the next 14 bytes:
             0x04, // type 4, not PACKET_COMPRESSED
-            0x66, 0x6F, 0x78, 0x20, 0x6A, 0x75, 0x6D, 0x70, 0x73, 0x20, 0x6F, 0x76, 0x65, // "fox jumps ove"
+            0x66, 0x6F, 0x78, 0x20, 0x6A, 0x75, 0x6D, 0x70, 0x73, 0x20, 0x6F, 0x76, 0x65, // "fox jumps over"
```

Comment:
Actually, it seems this is not a typo in the comment, the payload on the left appears to be indeed holding "fox jumps ove" without "r" :joy: